### PR TITLE
Update understanding-applocker-allow-and-deny-actions-on-rules.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4264
Added wording for clarification.

### CODE REVIEW

**Comment by jvsam** [comment]
Hi @nenonix and @JohanFreelancer9 for copy review please. Thank you.

**Comment by jvsam** [comment]
Hi @lindspea @nenonix and @JohanFreelancer9 from a perspective of a regular user reading the article, the amendments are still confusing for me. So basically, the default behavior is to 'deny action' unless explicitly added in the 'allowed list.' As stated in the article, *the **AppLocker rule collection** functions as an allowed list of files*, since by default, it's set to deny permission and you need to grant allow permission to override it. Anyway, looking forward to suggestions and updates. Thank you.

File: windows/security/threat-protection/windows-defender-application-control/applocker/understanding-applocker-allow-and-deny-actions-on-rules.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ This topic explains the differences between allow and deny actions on AppLocker
 
 ## Allow action versus deny action on rules
 
-Unlike Software Restriction Policies (SRP), each AppLocker rule collection functions as an allowed list of files. Only the files that are listed within the rule collection are allowed to run. This configuration makes it easier to determine what will occur when an AppLocker rule is applied.
+Unlike Software Restriction Policies (SRP), in a block by default, allow by exception configuration, each AppLocker rule collection functions as an allowed list of files. Only the files that are listed within the rule collection are allowed to run. This configuration makes it easier to determine what will occur when an AppLocker rule is applied.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-application-control/applocker/understanding-applocker-allow-and-deny-actions-on-rules.md]:
```suggestion
Unlike Software Restriction Policies (SRP) each AppLocker rule collection functions as an allowed list of files. Only the files that are listed within the rule collection are allowed to run. This **block by default, allow by exception** configuration makes it easier to determine what will occur when an AppLocker rule is applied.
```

**Comment by illfated** [comment]
Not sure what I have missed out on regarding comma use in English grammar, but I would like to see the comma reinserted in the sentence
"Unlike Software Restriction Policies (SRP), each AppLocker rule collection functions as an allowed list of files."

To my mind, it should be indifferent whether the clause "unlike Software Restriction Policies (SRP)" is added at the end or beginning, as long as the comma is used as a separator between the clause and the main sentence. (I could be wrong, though.)

File: windows/security/threat-protection/windows-defender-application-control/applocker/understanding-applocker-allow-and-deny-actions-on-rules.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ This topic explains the differences between allow and deny actions on AppLocker
 
 ## Allow action versus deny action on rules
 
-Unlike Software Restriction Policies (SRP), each AppLocker rule collection functions as an allowed list of files. Only the files that are listed within the rule collection are allowed to run. This configuration makes it easier to determine what will occur when an AppLocker rule is applied.
+Unlike Software Restriction Policies (SRP) each AppLocker rule collection functions as an allowed list of files. Only the files that are listed within the rule collection are allowed to run. This **block by default, allow by exception** configuration makes it easier to determine what will occur when an AppLocker rule is applied.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-application-control/applocker/understanding-applocker-allow-and-deny-actions-on-rules.md]:
```suggestion
Unlike Software Restriction Policies (SRP), each AppLocker rule collection functions as an allowed list of files. Only the files that are listed within the rule collection are allowed to run. This **block by default, allow by exception** configuration makes it easier to determine what will occur when an AppLocker rule is applied.
```

**Comment by jvsam** [comment]
Hi @mjcaparas this pull request has been copy edited and is ready for your final review and approval. Please let us know if there are outstanding issues to be resolved prior to merge. Thank you. See: [commits](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4344/commits) and [changes](https://github.com/MicrosoftDocs/windows-itpro-docs/pull/4344/files).

**Comment by jvsam** [comment]
Hi @mjcaparas thanks for approving this pull request. Can you also merge please? Thank you.

**Comment by jvsam** [comment]
Thank you so much @mjcaparas for merging this PR.

**Pull request merged**